.github/workflows/cicd-deploy-to-dev.yml

```
@@ -74,5 +75,6 @@ jobs:
             --set app.deployment.env.POSTGRES_DB=${POSTGRES_DB} \
             --set app.deployment.env.POSTGRES_HOST=${POSTGRES_HOST} \
             --set app.deployment.env.POSTGRES_PORT=${POSTGRES_PORT} \
+            --set app.deployment.env.ADMIN_GITHUB_TOKEN=${ADMIN_GITHUB_TOKEN} \
```

```suggestion
            --set app.deployment.env.READ_GITHUB_TOKEN=${READ_GITHUB_TOKEN} \
```
subsequent hair splitting